src/base/utils/misc.h

```diff
@@ -32,8 +32,13 @@
 #define UTILS_MISC_H
 
 #include <ctime>
+#include <memory>
 #include <vector>
 
+#ifdef Q_OS_WIN
+#include <cwchar>
```

Comment:
Here too?

Reply:
Thanks for double confirming this, I found:
>In C++, wchar_t is a distinct fundamental type (and thus it is not defined in <cwchar> nor any other header).

So I'll remove this.